server/config/config.go

```diff
@@ -18,7 +18,7 @@ const (
 
 // GetDefaultAPINamespaces returns the default list of JSON-RPC namespaces that should be enabled
 func GetDefaultAPINamespaces() []string {
-	return []string{"eth"}
+	return []string{"eth", "net", "web3"}
```

Comment:
also txpool ?

Reply:
The idea with this change is for most of them to be disabled so the node would not exposed any functionality that it explicitly want. Txpool's endpoints are just mocks at the moment so its not a big deal if they are enable but that was the point of the change.